com.valvesoftware.Steam.Utility.InhibitScreensaver.metainfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 trilader -->
+<component type="addon">
+  <id>com.valvesoftware.Steam.Utility.InhibitScreensaver</id>
+  <extends>com.valvesoftware.Steam</extends>
+  <name>InhibitScreensaver</name>
+  <developer_name>trilader</developer_name>
+  <summary>A small utility to run your applications and games with to enable screensave blocking from within flatpak</summary>
+  <description>
+    <p>A small utility to run your applications and games with to enable screensave blocking from within flatpak.</p>
+    <p>To use just put <pre>inhibit-screensaver %command%</pre> in your game/application launch command after installing this.</p>
+  </description>
+  <categories>
+    <category>Game</category>
+  </categories>
+  <icon height="128" width="128">inhibit-screensaver.svg</icon>
```

Comment:
```suggestion
```

Reply:
I removed the icon. Again for clarification: Did I use the icon tag incorrectly or is it just out of place for an extension package? I assume it would be listed in a extensions list somewhere (either when already installed or in an "app-store" type situation) which *could* display an icon, e.g. in a list view.